apps/client/src/components/PhoneNumber.tsx

```diff
@@ -28,10 +28,10 @@ export default ({
   <Wrapper>
     {link ? (
       <Link
+        action={actions.CLICK_PHONE_LINK}
+        eventName={`${eventNames.PHONE_NUMBER} - ${eventLocation}`}
```

Comment:
Dash usage seems not consistent. Sometimes it's there (` - `) sometimes it's not.

Reply:
That's true. Because sometimes we don't need a dash. We might look in to the array option. I will put in on the refactor list